src/System.IO.FileSystem/tests/FileSystemTest.cs

```diff
@@ -21,11 +21,22 @@ protected static bool CanCreateSymbolicLinks
         {
             get
             {
+                // Verify file symlink creation
                 var path = Path.GetTempFileName();
                 var linkPath = path + ".link";
                 var ret = MountHelper.CreateSymbolicLink(linkPath, path, isDirectory: false);
+                ret = ret && File.Exists(linkPath);
```

Comment:
In what situation will CreateSymbolicLink return true but the link won't have been created?